src/styles/normalize.scss

```diff
@@ -0,0 +1,8 @@
+* {
```

Comment:
Let's move your normalize file to the utils folder

Reply:
done)